README.md

```diff
@@ -137,44 +121,192 @@ export const MyAdmin = () => (
 );
 ```
 
-## Full Text Search Support
+## Features
```

Comment:
Don't you want to mention the FTS as one of the features too?

Reply:
No as there is nothing specific to mention. It's just postgrest